### ISSUE TITLE
Wykasowana pamięć

### ISSUE THREAD

**Issue by ramir3zkrk** [ramicspa/kodi#633]
Bardzo fajne repo i dobrze działało, ale pomimo tego, że nie miałem żadnego builda zainstalowanego (kupiłem Smart TV Box X96 dopiero wczoraj) i tak wyskoczył mi błąd i wykasowało dane.

**Comment by tomasiek** [comment]
Na tym boxie na 100% miałeś chińskiego builda. Na każdym z boxów zawsze jest full buildopodobnych ustrojstwo. Trzeba go usunąć, zainstalować czyste kodu ze sklepu Google Play lub z oficjalnej strony kodi.

**Comment by ramir3zkrk** [comment]
Skurw... cholery jedne! No patrz cholera jak to człowiek nie wie, a potem oskarżeniami rzuca na lewo i prawo. Dziękuję, tomasiek. Czyli zwyczajnie wejść do sklepu play, odinstalować francę i zainstalować na nowo, czy muszę jeszcze coś wyrzucać?

**Comment by tomasiek** [comment]
Nie. To co masz zainstalowane to tak napraawdę jest "chińskie kodi" - nie wiadomo co tam jest. Odinstaluj to przez androida - przez ustawienia/aplikacje a następnie wejdź normalnie na sklep play i zainstaluj kodi.
https://play.google.com/store/apps/details?id=org.xbmc.kodi
